node_agent/BUILD

```diff
@@ -0,0 +1,18 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_binary")
+
+go_library(
+    name = "go_default_library",
+    srcs = [ "main.go",
+    ],
+    visibility = ["//visibility:public"],
+    deps = [
+        "//node_agent/na:go_default_library",
```

Comment:
The directory name is related to the package name of the files under that  directory. This is the convention followed by other repos too. The "na" is short form for node agent and I thought it made sense. But if you don't think so, I am open to suggestions.